backend/datasource.go

```diff
@@ -91,15 +91,24 @@ func (ds *CassandraDatasource) MetricQuery(tsdbReq *datasource.DatasourceRequest
 
 		var created_at time.Time
 		var value float64
+
+		ds.logger.Debug(fmt.Sprintf("filtering: %s\n", queryData.Get("filtering").MustString()))
```

Comment:
This is too much because we already know filtering status from the next debug output